install codeceptjs versi 2.3.3 get respond error "cannot find module './utils"

I am install codeceptjs new version and follow step by step install "https://codecept.io/quickstart"
but get respond error " cannot find module './utils'".

step by step :

- npm init -y
- yarn add codeceptjs puppeteer --save-dev
- npx npx codeceptjs init
   geeting respond = cannot find module './utils'


maybe codeceptjs versi 2,3,3 is corrupt?

@pablopaul my functional test's error log told me which line throw error

```diff
Error: Cannot find module './utils'
     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
     at Function.Module._load (internal/modules/cjs/loader.js:506:25)
     at Module.require (internal/modules/cjs/loader.js:636:17)
     at require (internal/modules/cjs/helpers.js:20:18)
-    at Object.<anonymous> (/xxx/node_modules/parse-function/dist/cjs/index.js:1:731)
     at Module._compile (internal/modules/cjs/loader.js:688:30)
     at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
     at Module.load (internal/modules/cjs/loader.js:598:32)
     at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
     at Function.Module._load (internal/modules/cjs/loader.js:529:3)
```

I have been [report an issue][] to parse-function's repo, hope this issue can be fixed ASAP.

[report an issue]: https://github.com/tunnckoCore/parse-function/issues/202

--- a/package.json
+++ b/package.json
@@ -59,7 +59,7 @@
     "mocha": "^6.2.0",
     "mocha-junit-reporter": "^1.23.1",
     "ms": "^2.1.2",
-    "parse-function": "^5.2.10",
+    "parse-function": "^5.2.11",
     "promise-retry": "^1.1.1",
     "requireg": "^0.1.8",
     "resq": "^1.6.0",

